ptsites/base/work.py

```diff
@@ -1,12 +1,12 @@
-from typing import Union
+from typing import Union, Callable, Optional
 
 
 class Work:
-    def __init__(self, url: str = None, method: callable = None, data=None,
-                 succeed_regex: list[Union[str, tuple]] = None, fail_regex: str = None,
-                 assert_state: tuple = None, response_urls=None, use_last_content=False, is_base_content=False,
-                 **kwargs):
-        self.url: str = url
+    def __init__(self, url: Optional[str] = None, method: Optional[Callable] = None, data: Optional[dict] = None,
```

Comment:
显示地写为包含None是[PEP 484](https://peps.python.org/pep-0484/#union-types:~:text=A%20past%20version,be%20made%20explicit.)的要求：
> A past version of [this PEP](https://peps.python.org/pep-0484/#union-types) allowed type checkers to assume an optional type when the default value is None, as in this code:
> ```python
> def handle_employee(e: Employee = None): ...
> ```
> This would have been treated as equivalent to:
> ```python
> def handle_employee(e: Optional[Employee] = None) -> None: ...
> ```
> This is no longer the recommended behavior. Type checkers should move towards requiring the optional type to be made explicit.

Reply:
@madwind